docs/DeveloperGuide.md

```diff
@@ -364,98 +364,84 @@ c) they do not remember which field they want to search.
 ### 5.6 Meet Feature
 
 The `meet` feature allows the user to schedule meetings having an `Agenda`, a `Meeting Time`, a `Meeting Place`, and 
-`Meeting Attendees`. 
+`Meeting Attendees`. The `meet` command can only be issued from the `Home Page`. It is facilitated by such classes as
+`HomePageParser`, `MeetCommandParser`, and `UpdateCommand`.
```

Comment:
Not sure if you want to add MeetCommand to this list since its in the diagram

Reply:
Whoops